src/test-helpers.tsx

```diff
@@ -716,9 +716,15 @@ export const createFakeRef = (
 export const createStoreWithVersion = (
   /* istanbul ignore next */
   version: ExternalVersionWithDiff | ExternalVersionWithContent = fakeVersion,
```

Comment:
Since `version` is also optional, this function should use object parameters for everything. It will be quite a big diff to convert it all so I pushed that to a separate issue: https://github.com/mozilla/addons-code-manager/issues/965